docs/operator-manual/applicationset/Generators-Git.md

```diff
@@ -383,6 +383,15 @@ spec:
 
 In `values` we can also interpolate all fields set by the git files generator as mentioned above.
 
+## Wildcard Pattern
```

Comment:
@nitishfy, we already reference a dedicated page for this feature in the note at line 343. Maybe we could consider streamlining this section a bit, or just enhancing the existing reference to [this doc](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/applicationset/Generators-Git-File-Globbing.md), WDYT?

Reply:
Oh sorry, I completely missed that. Can we enable the new globbing pattern only for git file generator and not git directory generator as per [these docs](https://argo-cd.readthedocs.io/en/latest/operator-manual/applicationset/Generators-Git-File-Globbing/#enabling-the-new-globbing)?

Reply:
No need to apologize here 😅. The document seems to be specifically for Git File Generator. So as per those docs, it does give that sense.